pkg/apis/certificates/validation/validation.go

```diff
@@ -572,3 +579,263 @@ func validateTrustBundle(path *field.Path, in string) field.ErrorList {
 
 	return allErrors
 }
+
+// We don't care what you call your pod certificate requests.
+func validatePodCertificateRequestName(name string, prefix bool) []string {
```

Comment:
`NameIsDNSSubdomain` is easy and common.

Reply:
Why would we restrict this beyond normal Kubernetes naming restrictions?